Microsoft.DotNet.Try.Markdown/CodeFenceAnnotationsParser.cs

```diff
@@ -121,8 +133,36 @@ private Parser CreateOptionsParser(Action<Command> configureCsharpCommand = null
             csharp.AddAlias("CSHARP");
             csharp.AddAlias("cs");
             csharp.AddAlias("c#");
+            return csharp;
+        }
 
-            return new Parser(new RootCommand { csharp });
+        private static Command CreateFsharpCommand(Action<Command> configureCsharpCommand, Option packageOption,
+            Option packageVersionOption)
+        {
+            var fsharp = new Command("fsharp")
```

Comment:
Please refactor to avoid duplication. Anything that's shared between these commands would be better applied through a common method.